SimpleWAWS.Console/Program.cs

```diff
@@ -3,6 +3,7 @@
 using System;
```

Comment:
Yes. These are to recreate the resourcegroups to support functions/storage when we do the switchover. I can keep the existing code as is, and add a new method for re-creating the resourcegroups.